vro-types/o11n-plugin-vc/index.d.ts

```diff
@@ -15833,6 +15833,37 @@ declare class VcVirtualMachineBackupEventInfo {
 	constructor(eventType: string, code: number, message: string);
 }
 
+/**
+ * Provice information about storage
+ */
+declare class VcStorageQueryManager {
+  readonly vimType: string;
+  readonly sdkConnection: VcSdkConnection;
+  readonly moref: VcManagedObjectReference;
+  readonly vimId: string;
+  readonly sdkId: string;
+  readonly vimHost: VcSdkConnection;
+
+  /**
+   * create a trigger
+   * @param {number} timeout
+   * @param {string} filter
+   * @param {string} condition
+   * @param {string} filterToSync
+   * @returns {Trigger}
+   */
+  public createTrigger (timeout: number, filter: string, condition: string, filterToSync: string): Trigger;
+
+  /**
+   * get the hosts with a lun attached
+   * @param {string} arg0
+   * @returns {VcHostSystem[]} - list of ESX hosts
+   */
+  public queryHostsWithAttachedLun (arg0: string): VcHostSystem[];
+
+  constructor(vimType: string, sdkConnection: VcSdkConnection, moref: VcManagedObjectReference, vimId: string, sdkId: string, vimHost: VcSdkConnection);
```

Comment:
Are you sure this object has a constructor? I don't see it in the API explorer (but have it mind that i am not with latest version so mine might be outdated)